Dockerfile

```diff
@@ -0,0 +1,55 @@
+# First, get the elixir dependencies within an elixir container
+FROM elixir:1.11.3 AS elixir-builder
+
+ENV LANG="C.UTF-8" MIX_ENV=prod
+
+WORKDIR /root
+ADD . .
+
+# Install hex, rebar, and deps
+RUN mix local.hex --force && \
+  mix local.rebar --force && \
+  mix do deps.get --only prod
+
+# Next, build the frontend assets within a node.js container
+FROM node:14.15.5 as assets-builder
```

Comment:
Use a `buster`-based image here too, and consider omitting the point release so that you get minor updates automatically.